packages/outbox-execute/package.json

```diff
@@ -13,7 +13,7 @@
     "hardhat": "^2.9.1"
   },
   "dependencies": {
-    "@arbitrum/sdk": "^2.0.0",
+    "@arbitrum/sdk": "^2.0.9",
```

Comment:
can you search and replace other package.json, do a yarn install at root and push the yarn.lock?

Reply:
saw the other PR